complexify-android/src/main/java/co/infinum/complexify/ComplexityListener.kt

```diff
@@ -0,0 +1,15 @@
+package co.infinum.complexify
+
+/**
+ * Password complexity calculation listener
+ */
+interface ComplexityListener {
```

Comment:
I would suggest to change this to a SAM functional interface. Just add fun in front of it and watch the magic in Kotlin and Java 8 setup. You lose nothing, you gain a lot on readability.

Reply:
What a neat trick 👍